src/server/browserContext.ts

```diff
@@ -380,6 +383,21 @@ export abstract class BrowserContext extends SdkObject {
     this.on(BrowserContext.Events.Page, installInPage);
     return Promise.all(this.pages().map(installInPage));
   }
+
+  async startVideoRecording(options: types.VideoOptions) {
+    if (this._videoOptions)
+      await this.stopVideoRecording();
```

Comment:
This will also stop videos started via context option, won't it?

Reply:
Context option is mutually exclusive with tracing video.